spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/service/impl/DefaultSchedulerService.java

```diff
@@ -214,6 +215,23 @@ public void schedule(String scheduleName, String taskDefinitionName, Map<String,
 				.collect(Collectors.toMap(kv -> kv.getKey().substring(prefix.length()), kv -> kv.getValue()));
 	}
 
+	private List<String> tagCommandLineArgs(List<String> args) {
+		List<String> taggedArgs = new ArrayList<>();
+
+		for(String arg : args) {
+			if(arg.contains("spring.cloud.task.name")) {
+				continue;
+			}
+			String updatedArg = arg;
+			if (!arg.startsWith(DATA_FLOW_URI_KEY) && !"--".concat(arg).startsWith(DATA_FLOW_URI_KEY)) {
+					updatedArg = COMMAND_ARGUMENT_PREFIX +
+							taskConfigurationProperties.getTaskLauncherPrefix() + arg;
```

Comment:
We need to add the explicit dot here (Given we now have the task launcher prefix without the dot). Will fix when merging.